### PR TITLE
arm simulation

### PR DESCRIPTION
**Pull request by greenblitz4590**
😀👍

### CODE REVIEW

File: src/main/java/edu/greenblitz/robotName/OI.java [hunk 1]
```diff
@@ -46,7 +48,58 @@ public SmartJoystick getSecondJoystick() {
...
+                    public void execute() {
+                        wrist.setPower(-0.2);
+                    }
+
+                    @Override
+                    public void end(boolean interrupted) {
+                        wrist.setPower(0);
+                    }
+                }
+        );
```
Comment (NoamRosenberg08) [inline src/main/java/edu/greenblitz/robotName/OI.java:102]: delete

File: src/main/java/edu/greenblitz/robotName/subsystems/arm/Elbow.java [hunk 2]
```diff
@@ -96,5 +100,14 @@ public boolean isAtAngle(Rotation2d targetHeight) {
         return Math.abs(targetHeight.getRadians() - getAngleInRadians()) <= ElbowConstants.TOLERANCE;
     }
 
+    public Pose3d getPose3D (){
+        return new Pose3d(
+                new Translation3d(
+                        0,0.1,0.6
+                ),
```
Comment (NoamRosenberg08) [inline src/main/java/edu/greenblitz/robotName/subsystems/arm/Elbow.java:107]: move to a constant of PIVOT_TO_ROBOT_TRANSLATION

Comment (YoniKiriaty) [reply src/main/java/edu/greenblitz/robotName/subsystems/arm/Elbow.java:107]: its elbow and wrist btw

File: src/main/java/edu/greenblitz/robotName/OI.java [hunk 3]
```diff
@@ -27,7 +29,7 @@ private OI() {
         mainJoystick = new SmartJoystick(RobotConstants.Joystick.MAIN);
         secondJoystick = new SmartJoystick(RobotConstants.Joystick.SECOND);
         initButtons();
-        initializeDefaultCommands();
+//        initializeDefaultCommands();
```
Comment (bossyshrimp68) [inline src/main/java/edu/greenblitz/robotName/OI.java:32]: give it back

File: src/main/java/edu/greenblitz/robotName/subsystems/arm/Elbow.java [hunk 4]
```diff
@@ -96,5 +100,12 @@ public boolean isAtAngle(Rotation2d targetHeight) {
         return Math.abs(targetHeight.getRadians() - getAngleInRadians()) <= ElbowConstants.TOLERANCE;
     }
 
+    public Pose3d getPose3D (){
+        return new Pose3d(
+                ElbowConstants.ELBOW_POSITION_RELATIVE_TO_ROBOT,
+                new Rotation3d(-getAngleInRadians(),0, 0)
```
Comment (bossyshrimp68) [inline src/main/java/edu/greenblitz/robotName/subsystems/arm/Elbow.java:106]: magic minus??

**Review by bossyshrimp68 — CHANGES_REQUESTED** [review]

File: src/main/java/edu/greenblitz/robotName/subsystems/arm/Elbow.java (hunk 4 above)
Comment (YoniKiriaty) [reply src/main/java/edu/greenblitz/robotName/subsystems/arm/Elbow.java:106]: no its needed

**Review by aaa-png — APPROVED** [review]

**Review by NoamRosenberg08 — APPROVED** [review]

**Review by tnrz135 — APPROVED** [review]

**Pull request merged**